RU: add links to redis university courses/learning paths (#1093)
redis/docs: content/operate/rc/security/vpc-peering.md

{{< note >}}
{{< embed-md "rc-vpc-peering-cidr-list.md"  >}}
{{< /note >}}

## Continue learning with Redis University

> See the [Redis Cloud networking course](https://university.redis.io/course/xkz1coqj9tsqjr) for modules.

Reviewer: ```suggestion
See the [Redis Cloud networking course](https://university.redis.io/course/xkz1coqj9tsqjr) to learn more.
```
